app/services/process_feed_service.rb

```diff
@@ -230,6 +231,10 @@ def content(xml = @xml)
       xml.at_xpath('./xmlns:content', xmlns: TagManager::XMLNS).content
     end
 
+    def content_language(xml = @xml)
+      xml.at_xpath('./xmlns:content', xmlns: TagManager::XMLNS)['xml:lang']&.presence || 'en'
```

Comment:
Should that be hardcoded to `en` as default, or should it use whatever `I18n.default_locale` is?